CHANGES.rst

```diff
@@ -1,6 +1,46 @@
 Changes
 =======
 
+0.7.0 (TBD)
+-----------
+
+To upgrade, you need to set the following in your Scrapy project settings:
```

Comment:
- It is not technically necessary
- These Scrapy settings can be defined at the spider level

```suggestion
When upgrading, you should set the following in your Scrapy settings:
```